From 5a1c59a0329fda89df0dc8ca8497517dfc170c04 Mon Sep 17 00:00:00 2001
From: Yicong Yang <yangyicong@hisilicon.com>
Date: Tue, 27 Sep 2022 21:16:58 +0800
Subject: [PATCH] MAINTAINERS: Add maintainer for HiSilicon PTT driver
Patch-mainline: Queued in subsystem maintainer repository
Git-repo: https://git.kernel.org/pub/scm/linux/kernel/git/coresight/linux.git
Git-commit: 366317eae983a0d96aeed78ad219b9c4ed2a719a
References: bsn#12
Modified-by-SEL: No


Add maintainer for driver and documentation of HiSilicon PTT device.

Signed-off-by: Yicong Yang <yangyicong@hisilicon.com>
Reviewed-by: Jonathan Cameron <Jonathan.Cameron@huawei.com>
Link: https://lore.kernel.org/r/20220816114414.4092-6-yangyicong@huawei.com
Signed-off-by: Mathieu Poirier <mathieu.poirier@linaro.org>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 MAINTAINERS | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index d597d8fc67b1..1bc063c07158 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -7983,6 +7983,14 @@ F:	Documentation/admin-guide/perf/hisi-pcie-pmu.rst
 F:	Documentation/admin-guide/perf/hisi-pmu.rst
 F:	drivers/perf/hisilicon
 
+HISILICON PTT DRIVER
+M:      Yicong Yang <yangyicong@hisilicon.com>
+L:      linux-kernel@vger.kernel.org
+S:      Maintained
+F:      Documentation/ABI/testing/sysfs-devices-hisi_ptt
+F:      Documentation/trace/hisi-ptt.rst
+F:      drivers/hwtracing/ptt/
+
 HISILICON QM AND ZIP Controller DRIVER
 M:	Zhou Wang <wangzhou1@hisilicon.com>
 L:	linux-crypto@vger.kernel.org
-- 
2.34.1

